<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="produto">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>1080</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 25pt &quot;Courier&quot;;
background-color: rgb(253, 255, 216);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="empresa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;

background-color: rgb(223, 213, 255);

border-color: rgb(162, 162, 162);

border-style: Outset;
border-width: 2px;
border-radius: 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Users/Helio/.designer/backup/CADASTRO.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>420</width>
      <height>460</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 255, 216);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_fecha">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>610</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;F10&gt;FECHAMENTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Cliente:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sub_total">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>420</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 255, 216);
font: 700 19pt &quot;Courier&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>SUB-TOTAL:</string>
    </property>
   </widget>
   <widget class="QLabel" name="numero_pedido">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 255, 216);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de Desconto:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Desconto:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cabecalho">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>420</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 255, 216);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Forma de Pagamento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Previsao de Entraga:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Observacao:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="rb_percentual">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Percentual</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_valor">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Valor R$</string>
     </property>
    </widget>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_desconto">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDateEdit" name="data_previsao">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>350</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>1900</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_forma_pg">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_cliente">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>520</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="obs1">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>460</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="obs2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>440</y>
      <width>460</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Condicao de Pagamento:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia1">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia5">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia6">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia3_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia7">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia8">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia9">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia10">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia11">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia12">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia13">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia14">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dia15">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>450.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_entrada">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>20</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>20</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_qtd_dias">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>30</width>
      <height>40</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="obs3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>460</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="obs4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>520</y>
      <width>460</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="obs5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>560</y>
      <width>460</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_sair">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>610</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>bt_fecha</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
